d3a_api_client/redis_aggregator.py

```diff
@@ -199,7 +199,7 @@ def _on_tick(self, message):
 
     def _on_trade(self, message):
         logging.info(f"<-- {message.get('buyer')} BOUGHT {round(message.get('energy'), 4)} kWh "
-                     f"at {round(message.get('price'), 2)}/kWh -->")
+                     f"at {round(message.get('price'), 2)} cents/kWh -->")
```

Comment:
Price is in cents, not cents / kWh. If you want the energy rate you can either divide price / energy, or use the message["offer"]["energy_rate"] IIRC.